jscomp/syntax/tests/ppx/react/expected/typeConstraint.res.txt

```diff
@@ -1,27 +1,26 @@
 @@jsxConfig({version: 3})
 
 module V3 = {
-  @obj external makeProps: (~a: 'a, ~b: 'b, ~key: string=?, unit) => {"a": 'a, "b": 'b} = ""
+  @obj external makeProps: (~key: string=?, unit) => {.} = ""
```

Comment:
This change looks weird, why did a and b disappear?

Reply:
Good question: @mununki would you open an issue and take a look?
(Issue: `typeConstraint.res` does not look right in uncurried mode).
Alternatively, we just drop JSX V3 in compiler version 12.